src/components/Box/Box.tsx

```diff
@@ -16,6 +16,7 @@ export interface BoxProps
   > {
   ref?: React.Ref<HTMLDivElement>;
   className?: string;
+  flexChildren?: boolean;
```

Comment:
I was going to suggest that this be typed as typeof `flex` but I think we could add that later if there's a need, without it being a breaking change.